File: WPServer/WPProjectServer/WPProjectServer/Logic/SubsystemLogic/ISubsystemLogic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPProjectServer.Logic
{
    public abstract class ISubsystemLogic
    {
        LogicFactory logics;

        public ISubsystemLogic()
        {
            logics = new LogicFactory();
        }
    }
}

File: WPWeb/Library/src/main/java/com/logic/SubLogic/ISubsystemLogic.java
package com.logic.SubLogic;


import com.logic.LogicFactory;

/**
 * Created by Szymon on 2015-10-13.
 */
public abstract class ISubsystemLogic
{
    LogicFactory logicsFactory;

    public ISubsystemLogic(LogicFactory lgF)
    {
        logicsFactory = lgF;
    }
}
